app/helpers/image_encode_helper.rb

```diff
@@ -0,0 +1,33 @@
+module ImageEncodeHelper
+  def encodable_image_tag(source, options = {})
+    image_tag(encodable_image_source(source), options)
+  end
+  alias eimage_tag encodable_image_tag
```

Comment:
not a fan of "future convenience" reasons.  Would prefer we just name the method the way we want the API to look, and that should be the defined API.